libs/datamodel/core/tests/base/base_types.rs

```diff
@@ -238,3 +238,35 @@ fn resolve_enum_field() {
     role_enum.assert_has_value("PRO");
     role_enum.assert_has_value("USER");
 }
+
+#[test]
+fn json_list_type_must_work_for_some_connectors() {
+    let dml = indoc! {r#"
+        model User {
+          id   Int    @id
+          json_list Json[]
+        }
+    "#};
+
+    let error = datamodel::parse_schema(&format!("{}\n{}", COCKROACHDB_SOURCE, dml))
```

Comment:
Maybe use the `with_header` helper here instead. See example: https://github.com/prisma/prisma-engines/blob/master/libs/datamodel/core/tests/attributes/id_negative.rs#L551-L572

Reply:
Thanks. I've added in `with_header`